src/select/index.ts

```diff
@@ -434,13 +430,9 @@ export class Select extends ThemedMixin(FocusMixin(WidgetBase))<SelectProperties
 				hidden: labelHidden,
 				forId: widgetId
 			}, [ label ]) : null,
-			useNativeElement ? this.renderNativeSelect() : this.renderCustomSelect()
-		];
-
-		return v('div', {
-			key: 'root',
-			classes: this.theme(this.getRootClasses())
-		}, labelAfter ? children.reverse() : children);
+			useNativeElement ? this.renderNativeSelect() : this.renderCustomSelect(),
+			w(HelperText, { theme, text: helperText })
```

Comment:
Would you want to conditionally render the helper text widget? Something like `helperText ? w(HelperText, { theme, text: helperText }) : null`

Reply:
I wanted to render it at all times. The helpertext has both a root and a conditional text node. The reasoning behind this was to allow the consumer to theme it to reserve it's space if they wish in order to stop the UI jumping around if they helpertext shows / hides. In reality, it's best practise to always show a helperText for a widget that will be validated.